lib/movable_ink/aws/eks.rb

```diff
@@ -11,7 +11,9 @@ def eks(region: my_region)
 
       def generate_kubeconfig(region: my_region, cluster_name:)
         client = eks(region: region)
-        resp = client.describe_cluster({ name: cluster_name })
+        resp = run_with_backoff do
+          client.describe_cluster({ name: cluster_name })
+        end
         cluster_arn = resp.cluster.arn
```

Comment:
I think this is still going to blow up if no cluster is returned

Reply:
I changed it to return an empty string if there's no cluster. Let me know if you feel like it should behave differently